index.html

```diff
@@ -0,0 +1,46 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Mmakola Clive Portfolio</title>
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/font-awesome/4.7.0/css/font-awesome.min.css">
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <header class="header">
+        <ul class="toolbar">
+            <li class="name">
+                <a href="#">Mmakola Clive</a>
+            </li>
+            <li class="menu">
+                <img src="images/Union.png" alt="">
+            </li>
+        </ul>
```

Comment:
- [ ] The `<nav>` tag is used for declaring the navigational section in HTML documents. Websites typically have sections dedicated to navigational links, which enables users to navigate the site. For semantic reasons and other states above,  it would be better if you wrap your hamburger `menu button` into a `nav` tag.